packages/tse-backend/src/common/entities/market-making.entity.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import {
+  AmountChangeType,
+  PriceSourceType,
+  StrategyInstanceStatus,
+} from '../enums/strategy-type.enums';
+
+@Entity()
+export class MarketMaking {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  userId: string;
+
+  @Column()
+  clientId: string;
+
+  @Column()
+  pair: string;
+
+  @Column()
+  exchangeName: string;
+
+  @Column('decimal', { precision: 5, scale: 2 })
+  bidSpread: number;
+
+  @Column('decimal', { precision: 5, scale: 2 })
+  askSpread: number;
+
+  @Column('decimal', { precision: 10, scale: 2 })
+  orderAmount: number;
+
+  @Column()
+  checkIntervalSeconds: number;
+
+  @Column()
+  numberOfLayers: number;
```

Comment:
Check ehat's that

Reply:
Layers represents market-making orders placed at different price levels, with varying order amounts and spreads, around the current market price. This approach aims to increase liquidity and profit through frequent, small trades at multiple price levels.